code/src/components/Loading.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+
+export const Loading = () => {
+  const isLoading = useSelector((state) => state.ui.isLoading)
+  return (
+    <>{isLoading && <div>!! Loading !!</div>}</>
```

Comment:
We think that "isLoading" can be removed as you have conditionals for this in App.js.